cmake/UserOverride.cmake

```diff
@@ -17,21 +17,21 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
             set(common "${common} -DMS_WIN64")
         endif()
     endif()
```

Comment:
Lines 15-18, should belong in CMakeLists.txt

Reply:
Fixed in 100a52e07813ea54c7b3282adf733861dd09e27a.